### ISSUE TITLE
Compilation error in KRYPTO.ml

### ISSUE THREAD

**Issue by pirapira** [kframework/evm-semantics#135]
When I follow the instruction in `README` on Ubuntu 17.10, after `make` I see the following error

```
ocamlfind opt -o interpreter constants.cmx prelude.cmx plugin.cmx parser.cmx lexer.cmx run.cmx interpreter.ml -package gmp -package dynlink -package zarith -package str -package uuidm -package unix -package ethereum-semantics-plugin -linkpkg -inline 20 -nodynlink -O3 -linkall
File "KRYPTO.ml", line 38, characters 20-60:
Error: Unbound module Secp256k1.RecoverableSign
File "_none_", line 1:
Error: Cannot find file KRYPTO.cmx
```

I notice that the interface of `secp256k1` [changed a lot](https://github.com/dakk/secp256k1-ml/commit/5097deb195e0abf23b71e752b9f6ba3ac82fe5cc#diff-49135569748412a3baeabd3848be89cc) at 0.4.0.

**Comment by dwightguth** [comment]
I will investigate and fix this, but in the meantime the workaround should be to run `opam install secp256k1.0.3.2

**Comment by pirapira** [comment]
Thank you!

**Comment by dwightguth** [comment]
Fixed in #137

**Comment by pirapira** [comment]
Thanks a lot.
